evostar2019/angrybirds-evo19.tex

```diff
@@ -122,8 +124,40 @@
 %
 \section{Introduction}
 \label{sec:intro}
-\input{Context}
-\input{Motivation}
+%\input{Context}
```

Comment:
We are not doing this. That was created by the original author, and we are leaving it that way to preserve authorship.